app/views/admin/cms/_collection_list.html.erb

```diff
@@ -0,0 +1,41 @@
+<table>
+  <thead>
+    <tr>
+      <th>Collection ID</th>
+      <th>Title</th>
+      <th>Items</th>
+      <th>Published</th>
+      <th>Actions</th>
+      <th></th>
+    </tr>
+  </thead>
+
+  <tbody id="collection_list">
+    <% Collection.all.each do |resource| %>
```

Comment:
best to get into the habit of using `#find_each` unless good reason not to (actually, that inspires a question: do we want to add pagination to list pages?)